server/modules/turf-review/turf-review.service.ts

```
@@ -199,10 +200,85 @@ export default class TurfReviewService {
+        { $match: matchStage },
+        {
+          $group: {
+            _id: "$rating",
+            count: { $sum: 1 },
+          },
+        },
+      ])
+    ]);
+
+    // Fetch all reviews according to filter and sorting
+    let allReviews = await TurfReview.find(filter)
```

Biome (1.9.4)

[error] This let declares a variable that is only assigned once.

'allReviews' is never reassigned.

Safe fix: Use const instead.


(lint/style/useConst)